### PR TITLE
Add an age section to sops guide

### PR DESCRIPTION
**Pull request by souleb**
Explains how to use age to encrypt with a git workflow.
Links to the existing doc.

Signed-off-by: Soule BA <soule@weave.works>

### CODE REVIEW

File: content/en/docs/guides/mozilla-sops.md [hunk 1]
```diff
@@ -182,7 +182,13 @@ You can now commit the encrypted secret to your Git repository.
 Note that you shouldn't apply the encrypted secrets onto the cluster with kubectl. SOPS encrypted secrets are designed to be consumed by kustomize-controller.
 {{% /alert %}}
 
-### Using various cloud providers
+## Encrypting secrets using age
+
+age is a simple, modern alternative to OpenPGP. It's recommended to use age over PGP, if possible.
+
+Encrypting with age follows the same workflow than PGP. See the [Kustomization-controller API doc](https://github.com/fluxcd/kustomize-controller/blob/main/docs/spec/v1beta2/kustomization.md#age) for more details.
```
Comment (dholbach) [inline content/en/docs/guides/mozilla-sops.md:189]:
```suggestion
Encrypting with age follows the same workflow than PGP. See the [Kustomization-controller API doc](/docs/components/kustomize/kustomization/#age) for more details.
```

Should we maybe link to the same doc within the website instead?

File: content/en/docs/guides/mozilla-sops.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ Note that you should encrypt only the `data` or `stringData` section. Encrypting
 secret metadata, kind or apiVersion is not supported by kustomize-controller.
 {{% /alert %}}
 
-## Encrypt secrets
+## Encrypting secrets using PGP
```
Comment (stefanprodan) [inline content/en/docs/guides/mozilla-sops.md:161]:
```suggestion
## Encrypting secrets using OpenPGP
```

File: content/en/docs/guides/mozilla-sops.md [hunk 3]
```diff
@@ -182,7 +182,13 @@ You can now commit the encrypted secret to your Git repository.
 Note that you shouldn't apply the encrypted secrets onto the cluster with kubectl. SOPS encrypted secrets are designed to be consumed by kustomize-controller.
 {{% /alert %}}
 
-### Using various cloud providers
+## Encrypting secrets using age
+
+age is a simple, modern alternative to OpenPGP. It's recommended to use age over PGP, if possible.
```
Comment (stefanprodan) [inline content/en/docs/guides/mozilla-sops.md:187]:
```suggestion
[age](https://github.com/FiloSottile/age) is a simple, modern alternative to OpenPGP. It's recommended to use age over OpenPGP, if possible.
```

File: content/en/docs/guides/mozilla-sops.md (hunk 1 above)
Comment (souleb) [reply content/en/docs/guides/mozilla-sops.md:189]: Does this works? what I see in `../components/kustomize/_index.md` is a link to the controller docs.

Comment (stefanprodan) [reply content/en/docs/guides/mozilla-sops.md:189]: Change it like Daniel suggest then check if it works in the preview website that netlify creates.

Comment (souleb) [reply content/en/docs/guides/mozilla-sops.md]: great! 😊

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @souleb

**Pull request merged**